src/components/forms/ItemQuantityForm.tsx

```diff
@@ -3,6 +3,9 @@ import { ListItem } from "../../api";
 import { toast } from "react-hot-toast";
 import Button from "react-bootstrap/Button";
 import Form from "react-bootstrap/Form";
+import { FaEdit } from "react-icons/fa";
```

Comment:
Awesome use of these icons! These are the icons i was thinking when using the about page with this meger will have to update! Thanks for the reminder!